qtpy/QtGui.py

```diff
@@ -162,6 +162,19 @@ def movePositionPatched(
     QMouseEvent.globalPos = lambda self: self.globalPosition().toPoint()
     QMouseEvent.globalX = lambda self: self.globalPosition().toPoint().x()
     QMouseEvent.globalY = lambda self: self.globalPosition().toPoint().y()
```

Comment:
NB, these and those above (that I can't reach) should be removed in the merge, as they were replaced by the block added below.